### PR TITLE
docs: fix blkio-weight-device option args

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao mashimiao.fnst@cn.fujitsu.com

### CODE REVIEW

**Comment by thaJeztah** [comment]
docs changes LGTM

ping @sdurrheimer for a quick check of the completions (als @albers to check if the bash completions need to be fixed :heart:)

File: contrib/completion/zsh/_docker [hunk 1]
```diff
@@ -457,7 +457,7 @@ __docker_subcommand() {
         "($help -a --attach)"{-a=,--attach=}"[Attach to stdin, stdout or stderr]:device:(STDIN STDOUT STDERR)"
         "($help)*--add-host=[Add a custom host-to-IP mapping]:host\:ip mapping: "
         "($help)--blkio-weight=[Block IO (relative weight), between 10 and 1000]:Block IO weight:(10 100 500 1000)"
-        "($help)--blkio-weight-device=-[Block IO (relative device weight)]:device:Block IO weight: "
+        "($help)--blkio-weight-device=[Block IO (relative device weight)]:device:Block IO weight: "
```
Comment (sdurrheimer) [inline contrib/completion/zsh/_docker]: If `--blkio-weight-device` can be specified multiple times, you need to add a `*` in front of `--blkio-weight-device`.

```
"($help)*--blkio-weight-device=[Block IO (relative device weight)]:device:Block IO weight: "
```

**Comment by Mashimiao** [comment]
@sdurrheimer   thanks, fixed.

**Comment by thaJeztah** [comment]
Thanks for your contribution, @Mashimiao LGTM!

**Comment by thaJeztah** [comment]
PR that added support for this option is https://github.com/docker/docker/pull/13959

**Pull request merged**